extensions/sat/README.md

```diff
@@ -11,24 +11,41 @@ This document explains the fields of the Satellite Extension to a STAC Item. Sat
 The Satellite extension requires the [Instrument Fields](../../item-spec/common-metadata.md#instrument).
 
 - [Example (Landsat 8)](examples/example-landsat8.json)
+- [Example (Sentinel 1)](examples/example-sentinel1.json)
 - [JSON Schema](json-schema/schema.json)
 
 ## Item fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
+| sat:platform_serial_identifier | string | The International Designator, also known as COSPAR ID, and NSSDCA ID |
 | sat:orbit_state        | string        | The state of the orbit. Either `ascending` or `descending` for polar orbiting satellites, or `geostationary` for geosynchronous satellites |
+| sat:absolute_orbit     | integer       | The obsolute orbit number at the time of acquisition. |
 | sat:relative_orbit     | integer       | The relative orbit number at the time of acquisition. |
+| sat:anx_date_time     | date_time    | The Ascending Node Crossing (ANX) UTC time |
```

Comment:
I'm missing a description what this field is useful for.

Reply:
This is useful when you want to compute quickly orbital parameters without having to download the product. For instance to compute on the fly a baseline between 2 satellite acquisition This is used to find the best candidate in the archive from a post disaster event (e.g earthquake) scene acquisition for a DInSAR processing.

Reply:
@emmanuelmathot - could you just edit the PR to include more of that information? We want the description of 'sat:anx_date_time' to not just say what it is, but a bit of what it is used for. If it's more than a couple lines you can add more information below the table.

Reply:
Oh, I see now that you do have it below. Just put a suggestion in for how to edit it.